Component/PetList/PetList.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import pets from "@/Lib/Data";
+import Pet from "../Pet/Pet";
+
+const StyledPetList = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export default function PetList() {
+  return pets.length > 0 ? (
+    <StyledPetList>
+      {pets.map((pet, index) => {
+        return <Pet key={index} petData={pet} />;
+      })}
```

Comment:
Since your Data.js file contains a unique id for each pet, it's better to use id as the key instead of the index. 
Suggestion:  `{pets.map((pet) => {
  return <Pet key={pet.id} petData={pet} />;
})}`